department-of-veterans-affairs/va.gov-cms
# VACMS-5844: Install and configure Admin feedback module.

## Description

Closes #5811

## Testing done
manual, behat

## Screenshots
Front end:
![image](https://user-images.githubusercontent.com/8375335/165309895-3a226b22-7260-4043-8822-f681e0d533ed.png)

Dashboard: 
![image](https://user-images.githubusercontent.com/8375335/165310473-7db8b08d-23cb-4b5b-9914-24cc18c7aa64.png)


## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

As a cms user
1. Navigate to a knowledgebase page (`/help/cms-basics/how-to-log-inout`)
   - [x] Validate that the 'was this helpful' block appears at the bottom of the page
   - [x] Click a button and validate a thank you message appears
   - [x] If 'no' clicked, validate a message appears to go to help desk (this content is not 'final' yet)
2. Navigate to the dashboard (`/admin/content/feedback`)
   - [x] Validate the dashboard is available and has responses
   - [ ] View matches as many of our Views standards as possible, given the odd nature of this non-content type View.
3. Then validate Acceptance Criteria from issue
   - [x] We can collect metrics about each node, and available to knowledge base admins
   - [x] Add helpdesk link for 'no' responses 
   - [x] Style yes/no as secondary icon buttons 

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
